packages/trader/src/App/Components/Elements/ContractAudit/contract-details.tsx

```diff
@@ -75,6 +80,15 @@ const ContractDetails = ({
     const ticks_duration_text = isAccumulatorContract(contract_type)
         ? `${tick_passed}/${tick_count} ${localize('ticks')}`
         : `${tick_count} ${ticks_label}`;
+    const additional_info =
+        isResetContract(contract_type) && longcode ? (
+            <Localize
+                i18n_default_text='The reset time is {{ reset_time }}'
+                values={{ reset_time: longcode.split(/or\s+([^\n.]+)/i)[1] }}
```

Comment:
From `longcode` I'm extracting time, when the reset will be done (+-half of the contract length). Do not receive field for this. 
According to design, we should show some kind of notification in contract audit under Duration , like: 'Reset time is 2 ticks' (for 5 tick contract) or 'Reset time is 4 minutes' (for 8 minutes contract)

Reply:
@kate-deriv it's very bad we have to check the longcode because it's coming from BE and it can be changed, so we'll need to change the regex again.
is there no other API field we could take this data from? or can we request it?

Reply:
No, there is no other API filed.
`longcode` contains everything we need (see the screenshot, I just split on word 'or' and cut space), I've tested with different duration/longcode
May it be requested from BE as a future improvement? Otherwise it will block this card for another 5 month
<img width="269" alt="Screenshot 2023-10-31 at 17 06 17" src="https://github.com/binary-com/deriv-app/assets/121025168/ea2cbfe8-8409-485a-8b88-3fdd2cd1e296">

Reply:
Unfortunately, sometimes we rely on shortcode and extract something from it. It's a bad practice, but can we use longcode extraction while waiting for BE creating a new filed? This PR will be dead if it will wait more month

Reply:
We shouldn't have any logic in the code that requires to get text from BE and split/cut/replace/add or do any similar sorts of operations with it. Either design or API response needs to be adjusted.

Reply:
Moreover, it'll likely break if you switch the language at some point.

Reply:
@kate-deriv Please check how they do it in https://github.com/deriv-com/binary-static

Reply:
Yes, it's not working for other languages :( Will think how to modify